airflow/www/static/js/graph.js

```diff
@@ -172,8 +172,15 @@ function draw() {
       const task = tasks[nodeId];
       const tryNumber = taskInstances[nodeId].try_number || 0;
 
-      if (task.task_type === 'SubDagOperator') callModal(nodeId, executionDate, task.extra_links, tryNumber, true, dagRunId);
-      else callModal(nodeId, executionDate, task.extra_links, tryNumber, undefined, dagRunId);
+      callModal(
+        nodeId,
+        executionDate,
+        task.extra_links,
+        tryNumber,
+        task.task_tupe === 'SubDagOperator',
```

Comment:
```suggestion
        task.task_tupe === 'SubDagOperator' ? true : undefined,
```

Does it make a difference? This matches closer to the original.

Reply:
Yes it is, but it's not needed now because i changed how this parameter is handled